File: src/com/mrpowergamerbr/loritta/commands/vanilla/fun/CaraCoroaCommand.java
package com.mrpowergamerbr.loritta.commands.vanilla.fun;

import com.mrpowergamerbr.loritta.Loritta;
import com.mrpowergamerbr.loritta.commands.CommandBase;
import com.mrpowergamerbr.loritta.commands.CommandContext;

public class CaraCoroaCommand extends CommandBase {
	@Override
	public String getLabel() {
		return "girarmoeda";
	}

	@Override
	public void run(CommandContext context) {
		context.sendMessage(context.getAsMention(true) + (Loritta.getRandom().nextBoolean() ? "Cara!" : "Coroa!"));
	}

}

File: src/main/java/com/mrpowergamerbr/loritta/commands/vanilla/fun/CaraCoroaCommand.kt
package com.mrpowergamerbr.loritta.commands.vanilla.`fun`

import com.mrpowergamerbr.loritta.Loritta
import com.mrpowergamerbr.loritta.commands.AbstractCommand
import net.perfectdreams.loritta.api.commands.CommandCategory
import com.mrpowergamerbr.loritta.commands.CommandContext
import com.mrpowergamerbr.loritta.utils.locale.LegacyBaseLocale

class CaraCoroaCommand : AbstractCommand("flipcoin", listOf("girarmoeda", "flip", "toss", "moeda", "coin", "flipcoin"), CommandCategory.FUN) {
	override fun getDescription(locale: LegacyBaseLocale): String {
		return locale["CARACOROA_DESCRIPTION"]
	}

	override suspend fun run(context: CommandContext,locale: LegacyBaseLocale) {
		context.sendMessage(context.getAsMention(true) + if (Loritta.RANDOM.nextBoolean()) "<:cara:412586256409559041> **${locale["CARACOROA_HEADS"]}!**" else "<:coroa:412586257114464259> **${locale["CARACOROA_TAILS"]}!**")
	}
}
